code/API_definitions/device-status.yaml

```
@@ -543,170 +603,183 @@ components:
+      example: qs15-h556-rt89-1298
 
-    Event:
-      description: Generic event structure
+    CloudEvent:
+      description: The notification callback
       required:
-        - eventType
-        - eventTime
-        - eventDetail
+        - id
+        - source
+        - specversion
+        - type
+        - time
       properties:
-        eventType:
+        id:
+          type: string
+          description: identifier of this event, that must be unique in the source context.
+          example: sd5e-uy52-88t4-za66
+        source:
+          $ref: "#/components/schemas/Source"
+        type:
           type: string
-        eventTime:
-          $ref: '#/components/schemas/EventTime'
-        eventId:
+          description: "type of event as defined in each CAMARA API"
+          example: org.camaraproject.device-status.v0.roaming-status
+        specversion:
           type: string
-          format: uuid
-        eventDetail:
+          description: Version of the specification to which this event conforms (must be 1.0 if it conforms to cloudevents 1.0.2 version)
+          example: 1.0
+        datacontenttype:
+          type: string
+          description: 'media-type that describes the event payload encoding, must be "application/json" for CAMARA APIs'
+          example: application/json
+        data:
           type: object
+          description: Event details payload described in each CAMARA API and referenced by its type
+        time:
+          $ref: "#/components/schemas/DateTime"
       discriminator:
-        propertyName: 'eventType'
+        propertyName: "type"
         mapping:
-          ROAMING_STATUS: '#/components/schemas/EventRoamingStatus'
-          ROAMING_CHANGE_COUNTRY : '#/components/schemas/EventChangeCountry'
-          ROAMING_OFF: '#/components/schemas/EventRoamingOff'
-          ROAMING_ON: '#/components/schemas/EventRoamingOn'
-          SUBSCRIPTION_ENDS : '#/components/schemas/EventSubscriptionEnds'
+          org.camaraproject.device-status.v0.roaming-status: "#/components/schemas/EventRoamingStatus"
+          org.camaraproject.device-status.v0.roaming-change-country: "#/components/schemas/EventRoamingChangeCountry"
+          org.camaraproject.device-status.v0.roaming-off: "#/components/schemas/EventRoamingOff"
+          org.camaraproject.device-status.v0.roaming-on: "#/components/schemas/EventRoamingOn"
+          org.camaraproject.device-status.v0.subscription-ends: "#/components/schemas/EventSubscriptionEnds"
+
+    Source:
+      type: string
+      format: uri-reference
+      minLength: 1
+      description: "Identifies the context in which an event happened in the specific Provider Implementation."
+      example:
+        - https://github.com/cloudevents
+        - mailto:cncf-wg-serverless@lists.cncf.io
+        - urn:uuid:6e8bc430-9c3a-11d9-9669-0800200c9a66
+        - cloudevents/spec/pull/123
+        - /sensors/tn-1234567/alerts
+        - 1-555-123-4567
```

Here we should give one `example` or multiple `examples` coherent with examples of events (at the end of the file)
It can be:
- Absolute URI like: `https://notificationSendServer12.supertelco.com` starting with **uri-scheme**
- Relative Reference like:  `/supertelco/notification/SendServer12`
- 
```suggestion
      example: https://notificationSendServer12.supertelco.com
```